orchestra/todos/api.py

```diff
@@ -83,14 +82,13 @@ def _add_template_todo(
         additional_data):
     remove = _to_exclude(conditional_props, template_todo.get('remove_if', []))
     if not remove:
-        if parent_todo.skipped_datetime:
-            skipped_datetime = parent_todo.skipped_datetime
+        if parent_todo.status == Todo.Status.DECLINED.value:
+            to_skip = True
```

Comment:
lol was to_skip not getting used anywhere?

Reply:
I think it was used here: `skipped_datetime = timezone.now() if to_skip else None` (line 91)

Reply:
It was being used as @jumasheff pointed out.

Reply:
ah perfect! silly me! :)